web/.eslintrc.js

```diff
@@ -0,0 +1,105 @@
+module.exports = {
+  "extends": [
+    "airbnb-typescript/base"
+  ],
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "project": "tsconfig.json",
+  },
+  "plugins": [
+        "@typescript-eslint",
+        "@typescript-eslint/tslint"
+  ],
+  "rules": {
+    "arrow-parens": ["off", "as-needed"],
```

Comment:
where did you get the rules from? isn't the airbnb base good enough? Or is this just to deactivate the rules for now?

Reply:
Must rules are just deactivated to make the build run until they are fixed.
Some rules didn't matched our format but could be configured to match it.
(e.g. indent is 2 not 4)